iosApp/iosApp/Pages/StopDetails/StopDetailsFilteredView.swift

```diff
@@ -121,18 +112,22 @@ struct StopDetailsFilteredView: View {
             }
             .fixedSize(horizontal: false, vertical: true)
 
-            if let patternsByStop {
+            if !nearbyVM.groupByDirection {
+                DebugView { Text(String(
+                    stringLiteral: "Turn on the Group by Direction feature toggle, we're dropping support for headsign grouping"
```

Comment:
```suggestion
                    stringLiteral: "Turn on the Group by Direction feature toggle, we’re dropping support for headsign grouping"
```

I need to be stopped. The macOS keyboard shortcut for [typographically correct apostrophes](https://practicaltypography.com/apostrophes.html) is in my muscle memory, though, and once I saw this straight quote, I couldn’t unsee it.